CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+
+### Writing Code
+Please follow our [Python style guide](http://ateam-bootcamp.readthedocs.org/en/latest/reference/python-style.html), and also test with [pylama](https://pypi.python.org/pypi/pylama). If something is unclear, look at existing code which might help you to understand it better.
+
+### Submitting Patches
+When you think the code is ready for review, a pull request should be created on Github. Owners of the repository will watch out for new PR's and review them at regular intervals. For each update to the PR, we automatically run all the tests via [Travis CI](http://travis-ci.org/). If tests are failing, it's best to address the failures before requesting review. Otherwise you can wait for a review. If comments have been given in a review, they have to get integrated. For those changes, a separate commit should be created and pushed to your remote development branch. Don't forget to add a comment in the PR afterward, so everyone gets notified by Github. Keep in mind that reviews can span multiple cycles until the owners are happy with the new code.
+
+## Managing the Repository
+
+### Merging Pull Requests
+Once a PR is in its final state it needs to be merged into the upstream master branch. For that please **DO NOT** use the Github merge button! But merge it yourself on the command line. Reason is that we want to have a clean history. Before pushing the changes to upstream master, make sure that all individual commits have been squashed into a single one with a commit message ending with the issue number, e.g. "Fix for broken download behavior (#45)". Also check with `git log` to not push merge commits. Only merge PRs where Travis does not report any failure!
+
+### Versioning
+At irregular intervals, we release new versions of Firefox UI Tests. Therefore we make use of milestones in the repository. For each implemented feature or fix, the issue's milestone flag should be set to the next upcoming release. That way it's easier to see what will be part of the next release.
+
+When releasing a new version of Firefox UI ests please ensure to also update the history.md file with all the landed features and bug fixes. You are advised to use the [following issue](https://github.com/mozilla/firefox-ui-tests/issues/303) as template for the new release issue which needs to be filed. Please also check the associated PR for the code changes to be made.
```

Comment:
This versioning section would need a rewrite regarding our other branches. But given that our tests will land in the tree soon, I don't see a value in spending much time on it. So I would say just drop it.